apps/src/netsim/netsim.js

```diff
@@ -125,7 +133,7 @@ NetSim.prototype.init = function(config) {
    * Configuration for the loaded level
    * @type {netsimLevelConfiguration}
    */
-  this.level = config.level;
+  this.level = netsimUtils.scrubLevelConfiguration_(config.level);
```

Comment:
This is kludgy.  Is there a better way for me to bring in a "number or Infinity" value from levelbuilder, when `Infinity` is not a real value in JSON?